code/src/uarm_swift_pro.cpp

```diff
@@ -1,13 +1,14 @@
 #include <uarm_swift_pro.hpp>
 
 namespace r2d2::robot_arm {
-    uarm_swift_pro_c::uarm_swift_pro_c(const r2d2::hardware_usart_c &usart_bus)
+    uarm_swift_pro_c::uarm_swift_pro_c(usart_c &usart_bus)
         : usart_bus(usart_bus) {
     }
-    uarm_swift_pro_c::uarm_swift_pro_c(unsigned int &bautrate,
-                                       r2d2::uart_ports_c &usart_port)
-        : usart_bus(bautrate, usart_port) {
-    }
+
+    // uarm_swift_pro_c::uarm_swift_pro_c(unsigned int &bautrate,
+    //                                 r2d2::usart::uart_ports_c &usart_port)
```

Comment:
commented out code will not be approved when merging with the master branche.

Reply:
fixed in https://github.com/R2D2-2019/robot_arm/commit/654c3de55e13a57ff32a750c14c4098c7a6bed7b